### PR TITLE
MLB_Stats Scrapper

### LINKED ISSUE #4: Scrape MLB player stats
# Description

Write a script to scrape the stats of a user-defined MLB player and store them in CSV file.
You can use this website: [MLB Players](https://www.mlb.com/players)

## Type of issue

- [x] Feature (New Script)

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by Tomyzon1728**
# Description

A short summary of what is included in your Pull Request.

Fixes #4 

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by Tomyzon1728** [comment]
@AdityaJ7 and @chavarera , I've followed due process and my code works perfectly

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Please make a script which takes input from the user about the player whose stat they want to scarpe.
Don't use jupyter notebook.

Jupyter Notebook is good for analysis purpose, but here we want to make a dynamic script which can be used by others directly from the command line without making changes in the code.

Please read issue statement carefully.

Thanks

**Comment by Tomyzon1728** [comment]
Ok will make changes

**Review by chavarera — CHANGES_REQUESTED** [review]
Don't Add multiple out put files

Make a script instead of writing code in notebook

**Comment by Tomyzon1728** [comment]
Ok will do that before tomorrow

**Comment by Tomyzon1728** [comment]
@AdityaJ7 I've changed the jupyter notebook to a python script. Also, the issue was to convert MLB stats to a CSV file .
@chavarera made the required changes as well .

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Please add a requirements.txt file and add a screenshot of calling the script from the terminal  and remove the old screenshot that you provided

Also please comment your code in necessary areas and please divide your code into functions!

**Comment by AdityaJ7** [comment]
> @AdityaJ7 I've changed the jupyter notebook to a python script. Also, the issue was to convert MLB stats to a CSV file .
> @chavarera made the required changes as well .

@Tomyzon1728  Read the issue again carefully , it says the user of the script must enter the name of the player whose stat they want to scrape!!!!

You have hardcoded one player only

File: Scripts/Web_Scrappers/MLB_Scrapper/MLB.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import pandas as pd 
+import re 
+import requests 
+from bs4 import BeautifulSoup 
+url  = "http://www.espn.com/mlb/history/leaders/_/breakdown/season/year/2018/start/1"
+page = requests.get (url)
```
Comment (chavarera) [inline Scripts/Web_Scrappers/MLB_Scrapper/MLB.py:6]: Follow DRY(Don't Repeat Yourself) principal

write a separate function which perform following task(Single function)

```
Accept URL,
Create Request
Get Page Source
Convert Into Soup Object
Return soup object
```

**Review by chavarera — CHANGES_REQUESTED** [review]
Add different functions to perform specific tasks

**Comment by Tomyzon1728** [comment]
Ok will make necessary edit
Thanks guys

**Review by chavarera — CHANGES_REQUESTED** [review]
write a separate function which perform following task(Single function)

**Comment by chavarera** [comment]
@Tomyzon1728 are you working on it?

**Comment by AdityaJ7** [comment]
Will close this PR after 2 days @Tomyzon1728

**Pull request closed without merging**